app/models/conference.rb

```diff
@@ -81,6 +82,21 @@ def user_registered? user
   end
 
   ##
+  # Delete all EventSchedules that are not in the hours range
+  # After the conference has been successfully updated
+  def delete_event_schedules
+    if self.start_hour_changed? || self.end_hour_changed?
+      event_schedules = EventSchedule.select do |e|
```

Comment:
> Maybe adding a has_many :event_schedules to the conference.

Yes looks like the ideal solution. But it requires a lot of changes so should it be a part of this PR or should we handle this in a different PR? @Ana06 @lagartoflojo